src/Aframe/arcball-controller.js

```diff
@@ -90,24 +90,24 @@ THREE.TrackballControls = function (object, domElement) {
 
   // events
 
-  var changeEvent = { type: "change" };
-  var startEvent = { type: "start" };
-  var endEvent = { type: "end" };
+  let changeEvent = { type: "change" };
+  let startEvent = { type: "start" };
+  let endEvent = { type: "end" };
 
   // methods
 
   this.handleResize = function () {
```

Comment:
I'd swap this out for an arrow function to make sure the `this` is the right `this`